tests/integration/golang/aim/metric/search_test.go

```diff
@@ -162,11 +162,7 @@ func (s *SearchMetricsTestSuite) Test_Ok() {
 		RunID:     run2.ID,
 		Iter:      3,
 		Context: models.Context{
-			Json: datatypes.JSON([]byte(`
-				{
-					"testkey": "testvalue"
-				}`,
-			)),
+			Json: datatypes.JSON([]byte(`{"testkey":"testvalue"}`)),
```

Comment:
we probably don't need to have `datatypes.JSON([]byte`. under the hood `datatypes.JSON` is already `[]byte` so we can use just `datatypes.JSON` without `[]byte`.

Reply:
done